packages/@ember/-internals/glimmer/tests/integration/application/engine-test.js

```diff
@@ -875,7 +875,7 @@ moduleFor(
 
     ["@test query params don't have stickiness by default between model"](assert) {
       assert.expect(1);
-      let tmpl = '{{#link-to "blog.category" 1337}}Category 1337{{/link-to}}';
+      let tmpl = '{{#link-to "category" 1337}}Category 1337{{/link-to}}';
```

Comment:
So this is one way you can tell this is a breaking change. Do people really use engines without the engines addon though?

Reply:
I think it makes ember-engines more magic than expected, I like the way how ember-engines are explicit on route path

cc: @dgeb @rwjblue

Reply:
@chancancode I don't think we actually support engines without the ember-engines addon. All of our documentation and guidance on usage of engines has been in context of using the addon (roughly akin to how we now are allowed to assume folks use ember-source through ember-cli).

Reply:
@rwjblue and how about don't use `engine.route-name` anymore?